eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -485,6 +498,12 @@ steps:
     TargetFolder: "$(BuildOutputTargetPath)\\symbols-rtm"
   condition: " and(succeeded(), not(eq(variables['BuildRTM'], 'false')), eq(variables['IsOfficialBuild'], 'true')) "
 
+- task: PublishPipelineArtifact@1
+  displayName: "Publish symbols as pipeline artifacts"
+  inputs:
+    targetPath: "$(Build.Repository.LocalPath)\\artifacts\\symbolstoindex"
+    artifactName: "symbols - $(RtmLabel)"
```

Comment:
Should this match legacy `Copy Symbols` task only copy when Official Build is true?